### ISSUE TITLE
Setting `size` in attribute definition is not reflected in MySQL

### ISSUE THREAD

**Issue by shobhitsinghal624** [balderdashy/sails-mysql#159]
MySQL uses the default varchar(255) type even when a size is specified.

Sample model definition:

``` .js
// api/models/Pet.js
module.exports = {
  attributes: {
    uid:{
      type:'string',
      size: 36,
      unique: true,
    },
    name: {
      type: 'string',
    },
  }
}
```

MySQL table definition
```
mysql> desc pet;
+-------------------+--------------+------+-----+---------+-------+
| Field             | Type         | Null | Key | Default | Extra |
+-------------------+--------------+------+-----+---------+-------+
| id                | int(32)      | NO   | PRI | NULL    |       |
| uid               | varchar(255) | YES  |     | NULL    |       |
| name              | varchar(255) | YES  |     | NULL    |       |
| createdAt         | datetime     | YES  |     | NULL    |       |
| updatedAt         | datetime     | YES  |     | NULL    |       |
+-------------------+--------------+------+-----+---------+-------+
```

**Comment by sailsbot** [comment]
Thanks for posting, @shobhitsinghal624.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
